### PR TITLE
adding OCR and translation

### PR DESCRIPTION
**Pull request by behrica**
see #21

### CODE REVIEW

File: R/keyword_search.r [hunk 1]
```diff
@@ -76,26 +78,46 @@ keyword_search <- function(x, keyword, path = FALSE, split_pdf = FALSE,
...
+                           translate_fun = function(x,...) {return(x)},
+                           translate_fun_params = nil,
+                           translation_target_language="eng",
+                           ...) {
+  print(x)
   if(path) {
-    x <- pdftools::pdf_text(x)
+    path <- x
+    x  <- suppressMessages(pdftools::pdf_text(x))
+    output_dir_name <- paste0(dirname(path),"/outputs/")
```
Comment (behrica) [inline R/keyword_search.r:93]: My colleges found it important, that the tools is transparent on its three "magic tasks":
- text extraction from text
- OCR
- translation

So I write the results of this into up to three text files in an "output" folder.

**Comment by behrica** [comment]
Did you have the time to look at this PR ?

**Comment by lebebr01** [comment]
I have not yet, but hope to soon. I'll have to merge this into a new branch to explore why the build is not passing too.

**Comment by behrica** [comment]
I am not an expert in R packages,
so maybe I do something wrong with the declarations of methods to import / export.
And the best practices arround this.


If there is something for me to do, please let me know.

**Comment by lebebr01** [comment]
@behrica I made a few repo changes and in the process inadvertently closed this pull request. I deleted the master branch in favor of main, it said it would be changed over but apparently did not. 

Would you be able to create a new pull request and point it to the new ocr branch instead of the main branch?

**Pull request closed without merging**